dirty_memory_manager.cc

```diff
@@ -0,0 +1,176 @@
+// Copyright (C) 2012-present ScyllaDB
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+
+#include "dirty_memory_manager.hh"
```

Comment:
Does this patch just move code around or it also modifies it? The commit message is not entirely clear on this.

Reply:
It's plain code movement, but with namespace adjustment.